modules/ROOT/pages/architecture.adoc

```diff
@@ -42,6 +42,32 @@ The minimum requirements are 3 controller and 3 worker nodes. 3 controllers enab
 
 Mule applications run on workers. Multiple replicas of applications can run across workers.
 
+== Scaling Considerations
+
+* Required number of controller nodes. The maximum number of controller nodes supported is 3.
```

Comment:
Consider scaling the number of controller nodes when 
 - fault-tolernace is desired in case of a hardware failure on a controller node
 - production traffic is being served with applications running on Runtime Fabric

Reply:
Added this - thank you!